src/main/java/com/example/GymProject/dao/TraineeDao.java

```diff
@@ -92,13 +76,13 @@ public boolean containsKey(String key) {
     public void init() throws Exception {
         BufferedReader br = null;
         try {
-          logger.info("Starting Populating Trainee Storage");
-            File file = new File(traineeFilePath);
-            br = new BufferedReader(new FileReader(file));
+            logger.info("Starting Populating Trainee Storage");
```

Comment:
"Starting to populate Trainee Storage". Please avoid all uppercase starting letters for  messages. These messages are for you and dev team. And are just regular statements.